src/Uno.Extensions.Validation/Validator.cs

```diff
@@ -27,9 +27,9 @@ public Validator(IServiceProvider services)
 			try
 			{
 				context ??= new ValidationContext(instance);
-				System.ComponentModel.DataAnnotations.Validator.TryValidateObject(instance, context, results, true);
+				bool validates = System.ComponentModel.DataAnnotations.Validator.TryValidateObject(instance, context, results, true);
 
-				if (!results.Any() && instance is INotifyDataErrorInfo _instance)
+				if (!results.Any() && !validates && instance is INotifyDataErrorInfo _instance)
```

Comment:
Can you provide me with some context around this change? Based on the PR description I wouldn't expect to see this change.